python/taichi/lang/struct.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, *args, **kwargs):
 
         for k, v in self.__entries.items():
             if isinstance(v, (list, tuple)):
-                v = Matrix(v, ndim=matrix_ndim.get(k))
```

Comment:
It seems that `ndim` has no other usages so it can be removed from class `Matrix`.